recipe/meta.yaml

```
@@ -66,10 +68,11 @@ outputs:
         - cmake
         - make
         - llvm-openmp  # [osx]
-        - libgomp      # [linux]
+        - libgomp      # [linux or win]
       host:
         - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler != "None"]
         - nccl                                      # [linux and cuda_compiler != "None"]
+        - libgomp                                   # [win]
       run:
```

Think we need this in `run` as well

```suggestion
        - libgomp                                   # [win]
      run:
        - libgomp                                   # [win]
```

---

This is now fixed!  Hopefully we're ready to go now?